usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -297,8 +298,18 @@ def keyword_parse(keyword):
                 or_queryset |= Q(extent_competed__exact=v)
             queryset = queryset.filter(or_queryset)
 
-        elif key == "tas_codes":
-            queryset = build_tas_codes_filter(queryset, value)
+        # Because these two filters OR with each other, we need to know about the presense of both filters to know what to do
+        # This filter was picked arbitrarily to be the one that checks for the other
+        elif key == TasCodes.underscore_name:
+            if TreasuryAccounts.underscore_name in filters.keys():
+                q = TasCodes.build_tas_codes_filter(queryset, value)
+                q |= TreasuryAccounts.build_tas_codes_filter(queryset, filters[TreasuryAccounts.underscore_name])
+                queryset = queryset.filter(q)
+            else:
+                queryset = queryset.filter(TasCodes.build_tas_codes_filter(queryset, value))
```

Comment:
```suggestion
            q = TasCodes.build_tas_codes_filter(queryset, value)
            if TreasuryAccounts.underscore_name in filters.keys():
                q |= TreasuryAccounts.build_tas_codes_filter(queryset, filters[TreasuryAccounts.underscore_name])
            queryset = queryset.filter(q)

```

Reply:
Unless if these changes are unnecessary. Will any API endpoint trigger these matview_filters for old and new TAS?

Reply:
I don't know of any case where it would, but if we aren't deleting this file outright (which I'd rather not touch with this particular 10 foot pole) the code should be consistant.

Reply:
There is a ticket I am trying to have prioritized in Sprint 108 which will remove code made obsolete by the ES implementation. It's fine if you want to add the filter in this file.